app/assets/javascripts/controllers.js

```diff
@@ -17,6 +17,27 @@ Prometheus.Graph = {
     's': 1
   },
 
+  durationToString: function(seconds) {
+    // TODO(stuart): this is always returning seconds
+    // look to round to a certain amount based on the units
+    // in the graph? some level of precision lost.
```

Comment:
Can you explain this a bit more? If I understand you correctly, it's possible that the input is given in years, days, etc. (month is not possible?)? 

Where does this input come from? Is this a rickshaw thing or related to promdash?